### PR TITLE
Removed second value from "response"

### PR DESCRIPTION
**Pull request by Walter-Haydock**
As per conversation with @stevespringett - the response field should be exactly one item.

### CODE REVIEW

File: VEX/vex.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
       "analysis": {
         "state": "not_affected",
         "justification": "code_not_reachable",
-        "response": ["will_not_fix", "update"],
+        "response": "will_not_fix",
```
Comment (jkowalleck) [inline VEX/vex.json:86]: :x: per aplied JSON schema `response` is to be a list of known values. a LIST.
see also https://cyclonedx.org/docs/1.4/json/#vulnerabilities_items_analysis_response

**Review by jkowalleck — CHANGES_REQUESTED** [review]
Requested change would render the example to be an invalid document.

**Comment by Walter-Haydock** [comment]
Ok - thanks. I see that it is supposed to be a list now and must have missed it at first. I still remain confused by the example, though. How would a valid response be both "will_not_fix" and "update"? I could see "workaround_available" and "update" but the example as stands implies that the vendor will not fix the issue but also that they will.

Any clarification here would be helpful.

**Comment by stevespringett** [comment]
Common use case where a vendor states that an old EOL/EOS product is affected and that the customer should upgrade to a supported version.

**Pull request closed without merging**